src/agoradatatools/process.py

```diff
@@ -137,34 +147,32 @@ def process_dataset(
         )
         gx_runner.run()
 
-    syn_obj = load.load(
-        file_path=json_path,
-        provenance=dataset_obj[dataset_name]["provenance"],
-        destination=dataset_obj[dataset_name]["destination"],
-        syn=syn,
-    )
-
-    return syn_obj
```

Comment:
Why is the return statement removed but the function comment says it returns `syn_obj`?

Reply:
Because I forgot to edit the docstring 😅